chain/ethereum/src/block_stream.rs

```diff
@@ -173,6 +180,10 @@ where
                 start_blocks,
                 templates_use_calls,
                 metrics,
+
+                // A high number here forces a slow start, with a range of 1.
```

Comment:
Why don't we want to start fast? 😉

Reply:
Starting slow is more conservative. For example if we're coming from a large range size and suddenly hit a ton of triggers, causing the node to crash oom, starting slow gives us a chance to recover and find the correct range.

The range can increase by 10x on each scan, so it takes only 5 scans for this to hit the default maximum of 100k.

Reply:
Right, like when a node restarts and the subgraphs pick up at a range with an enormous amount of triggers. Ok, yes, makes sense. I was only thinking about the start case / genesis block, but with the start block feature, you could start at any block and too large a range could cause trouble.

Reply:
Ah yes there's that as well, when you start you don't know what your previous range was.